ui/package.json

```diff
@@ -68,6 +71,7 @@
     "reselect": "^4.0.0",
     "rimraf": "^3.0.2",
     "style-loader": "^2.0.0",
+    "ts-jest": "^26.5.4",
```

Comment:
Is `ts-jest` the only plugin needed to run tests on the `tsx` files (i.e., do we need any additional transpilers to handle the JSX?)

Reply:
Yep, that's it.

Reply:
sounds good!